examples/three-validators/hedera-setup.go

```diff
@@ -0,0 +1,129 @@
+/*
+* Copyright 2021 LimeChain Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/hashgraph/hedera-sdk-go"
+)
+
+func main() {
+
+	prKey := flag.String("prKey", "0x0", "Hedera Private Key")
+	accountID := flag.String("accountId", "0.0", "Hedera Account ID")
+	flag.Parse()
+
+	if *prKey == "0x0" {
+		panic("Private key was not provided")
+	}
+	if *accountID == "0.0" {
+		panic("Account id was not provided")
+	}
+
+	client := previewClient(*prKey, *accountID)
```

Comment:
I think that users should be able to choose the network using an argument `network=previewnet|testnet|mainnet`